src/bidiMapper/domains/context/PreloadScriptIdStorage.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import crypto from 'crypto';
+
+import {CommonDataTypes, Script} from '../../../protocol/protocol';
+
+export type PreloadScriptIdEntry = {
```

Comment:
It would make sense to split:
```
BiDiPreloadScript: {
  bidiId: UUID,
  browingContext: string | null,
  script: string,
  createdCdpPreloadScripts: CdpPreloadScript[]
}

CdpPreloadScript: {
  cdpId: string,
  cdpSessionId: string
}
```